hugo/layouts/_default/_markup/render-codeblock.html

```diff
@@ -0,0 +1,41 @@
+{{/*
+Creates a copy to clipboard button above each code block that switches between a copy and copied state
+by controlling the buttons visibility with js.
+
+If the code block is marked as citation style, placeholders inside the citation text will be replaced and
+the code block will be styled as a citation block.
+Otherwise, rendered as regular code block.
+*/}}
+
+{{- $content := .Inner | safeHTML -}}
+{{- $style := index .Attributes "style" | default "code" -}}
+
+<div class="clipboard">
+    <button id="copy-button" class="clipboard-button" type="button">
```

Comment:
The HTML specification demands that `id` attribute values [be unique](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) in the document. So maybe a class would be better.

Reply:
Good point, I've updated this in commit 2ea3400dccce2c949380bce104d2ad0aa6fcd730